src/EditorFeatures/Core/Implementation/TextStructureNavigation/AbstractTextStructureNavigatorProvider.TextStructureNavigator.cs

```diff
@@ -79,7 +79,7 @@ private TextExtent GetExtentOfWordWorker(SnapshotPoint position, CancellationTok
                     return new TextExtent(new SnapshotSpan(line.End, line.EndIncludingLineBreak - line.End), isSignificant: false);
                 }
 
-                var document = GetDocument(position, cancellationToken);
+                var document = GetDocument(position);
```

Comment:
is this just deleting an unused parameter to internal API?

Reply:
yup!

Reply:
see below, these are messages about unused parameters for things like private methods.  public stuff, or stuff that implements interfaces and the like aren't reported.